src/reversion_rest_framework/mixins.py

```diff
@@ -14,24 +18,48 @@ class BaseHistoryModelMixin:
 
 
 class HistoryOnlyMixin(BaseHistoryModelMixin):
+
+    def build_serializer(self, instance_class: type, queryset: QuerySet, many: Optional[bool] = False):
+        """wraps the original serializer within the Version serializer on the field_dict field
+        """
+
+        class _InstanceSerializer(ModelSerializer):
+
+            class Meta:
+                model = instance_class
+                fields = "__all__"
+
+        class _VersionsSerializer(self.version_serializer):
+            field_dict = SerializerMethodField()
+
+            @staticmethod
+            def get_field_dict(obj):
+                model_serializer = _InstanceSerializer(data=obj.field_dict)
```

Comment:
here we use the transient ModelSerializer to serialize the json blob returned from the version